src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1249,6 +1256,794 @@ subroutine eigencalving(domain, err)
+      real (kind=RKIND), dimension(:), pointer :: requiredCalvingVolumeRate
+      real (kind=RKIND), dimension(:), pointer :: calvingVelocity
+      real (kind=RKIND), dimension(:), pointer :: eMax, eMin, exx, exy, eyx, eyy
+      real (kind=RKIND), dimension(:,:), pointer :: flowParamA
+      real (kind=RKIND), dimension(:,:), pointer :: temperature
+      real (kind=RKIND), dimension(:), pointer :: floatingBasalMassBal
+      real (kind=RKIND), dimension(:), pointer :: damage
+      real (kind=RKIND), dimension(:), pointer :: ddamagedt
+      real (kind=RKIND), dimension(:), pointer :: szero, nstar, epsmax, visc, sigmamax
+      real (kind=RKIND), dimension(:), pointer :: source
+      real (kind=RKIND), dimension(:), pointer :: damage_nye
+      real (kind=RKIND), dimension(:), pointer :: damage_max
+      real (kind=RKIND), dimension(:), pointer :: stiffnessFactor
+      real (kind=RKIND), pointer ::  &
+           config_ice_density,            & ! ice density
+           config_ocean_density             ! ocean density
+
+      real (kind=RKIND), save :: rhoi    ! ice density (kg m^{-3}), copied from config_ice_density
+      real (kind=RKIND), save :: rhoo    ! ocean density (kg m^{-3}), copied from config_ocean_density
+
+      type (field1dReal), pointer :: meanFlowParamAField
+      real, dimension(:), pointer :: meanFlowParamA
+      !type (field1dReal), pointer :: viscField
+      !real, dimension(:), pointer :: visc
+      type (field1dReal), pointer :: sigmayxField
+      real, dimension(:), pointer :: sigmayx
+      type (field1dReal), pointer :: sigmaxyField
+      real, dimension(:), pointer :: sigmaxy
+      type (field1dReal), pointer :: sigmaxxField
+      real, dimension(:), pointer :: sigmaxx
+      type (field1dReal), pointer :: sigmayyField
+      real, dimension(:), pointer :: sigmayy
+      type (field1dReal), pointer :: sigmaminField
+      real, dimension(:), pointer :: sigmamin
+      !type (field1dReal), pointer :: sigmamaxField
+      !real, dimension(:), pointer :: sigmamax
+      type (field1dReal), pointer :: alphaField
+      real, dimension(:), pointer :: alpha
+      !type (field1dReal), pointer :: epsmaxField
+      !real, dimension(:), pointer :: epsmax
+      !type (field1dReal), pointer :: nstarField
+      !real, dimension(:), pointer :: nstar
+      !type (field1dReal), pointer :: szeroField
+      !real, dimension(:), pointer :: szero
+      !type (field1dReal), pointer :: sourceField
+      !real, dimension(:), pointer :: source
+      !type (field1dReal), pointer :: ddamagedtField
+      !real, dimension(:), pointer :: ddamagedt
+      type (field1dReal), pointer :: forceField
+      real, dimension(:), pointer :: force
+      type (field1dReal), pointer :: meanTField
+      real, dimension(:), pointer :: meanT
+
+      ! TZ: we can comment out the scratch field as above and redefine it in Registry.xml if we want to see it in the output nc file
```

Comment:
I'm confused what is going on here and with the commented out lines above.  If you want to make scratch field to be able to be output, you only need to change the 'scratch' declaration in Registry - you don't need to change anything in the code itself.

Reply:
Thanks. Now I removed this commented out lines and also those uncessary scratch field definitions